app/views/mailers/notify_mailer/channel_invite_email.text.erb

```diff
@@ -0,0 +1,3 @@
+You are invited to the DEV Connect channel <%= @membership.chat_channel.channel_name %>
```

Comment:
We should avoid references to "DEV" as a community name as we are trying to make an app more generalized so that it would be usable for other communities. You can use `ApplicationConfig["COMMUNITY_NAME"]` instead of the hardcoded "DEV"